Gems/Atom/RHI/DX12/Code/Source/RHI/AsyncUploadQueue.cpp

```diff
@@ -33,12 +33,20 @@ namespace AZ
             ID3D12DeviceX* dx12Device = device.GetDevice();
 
             m_copyQueue = CommandQueue::Create();
-            
+
+            // The async upload queue should always use the primary copy queue,
+            // but because this change is being made in the stabilization branch
+            // we will put it behind a define out of an abundance of caution, and
+            // change it to always do this once the change gets back to development.
+        #if defined(AZ_DX12_USE_PRIMARY_COPY_QUEUE_FOR_ASYNC_UPLOAD_QUEUE)
```

Comment:
@moudgils It may be worth making this the only supported path

@bosnichd did you notice any perf differences in dx12 windows with this switch enabled?

Reply:
yeah I will address this for dx12 and metal

Reply:
I haven't tried it with this enabled on Windows yet, but can do so. I'm mainly putting it behind a define out of an abundance of caution because it's in the stabilization branch.

Reply:
You wont notice much difference as I dont think we have scenario where we are stress using both the Copy queues.